ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.directive.js

```diff
@@ -79,7 +82,7 @@ export function specEditorDirective($rootScope, $templateCache, $injector, $sani
             model: '='
         },
         controller: ['$scope', '$element', controller],
-        template: template,
+        templateUrl: TEMPLATE_URL,
```

Comment:
While do do this, could you change it to match what the other directive do? https://github.com/apache/brooklyn-ui/blob/b04423b169e0d9a749ace79ccd446b4c6f3bb232/ui-modules/blueprint-composer/app/components/dsl-viewer/dsl-viewer.js#L35-L37

So it can be overridden directly from the attribute `template-url` on the directive itself.

Reply:
good spot, sorry i missed this!